### ISSUE TITLE
Cannot install windows agent using pe_bootstrap or pe_agent on PE2015.3

### ISSUE THREAD

**Issue by npwalker** [oscar-stack/vagrant-pe_build#110]
Trying to use pe_bootstrap fails to find a msi because the URL scheme appears to have changed 

```
An error occurred while trying to download the given resource (https://s3.amazonaws.com/pe-builds/released/2015.3.1/puppet-enterprise-2015.3.1-x64.msi),
and the transfer could not complete. The error message generated was "403 Forbidden".
```

Trying to use pe_agent drops back to using pe_bootstrap???  I set the version to current but it complains the version is unset.  

```
==> agent201531-server2008r2: Running provisioner: pe_agent...
==> agent201531-server2008r2: Installing Windows agent with PE Bootstrap
The Puppet Enterprise version must be set either on the global pe_build config
object or specified on a per-provisioner basis, but both were unset.
```

Setting a version for the puppet-agent results in an error:

```
There are errors in the configuration of this machine. Please fix
the following errors and try again:

pe_agent provisioner:
* The agent version 1.3.2 is too old; pe_agent can only provision versions
newer than 2015.2.0.
```

**Comment by Sharpie** [comment]
> Trying to use pe_bootstrap fails to find a msi because the URL scheme appears to have changed

I noticed this around the release of 2015.3.0 as well. It appears that the the backend URL scheme for Windows downloads may have changed and in a way that breaks the assumptions made by the `pe_bootstrap` installer.

For now, just download the MSI and move it to `~/.vagrant.d/pe_builds/puppet-enterprise-2015.3.0-x64.msi`.

> Trying to use pe_agent drops back to using pe_bootstrap???

That is currently the intended behavior. `pe_agent` runs simplified installations, but PE Simplified Agent installation doesn't support Windows agents. So, `pe_bootstrap` is used as a fallback.

**Comment by Sharpie** [comment]
Just confirmed that this is new in 2015.3.x. All 2015.2.x download URLs work as expected. Yay.

**Comment by Sharpie** [comment]
Turns out that pe_repo does stage windows .msi installers on the master starting with 2015.2 --- it just doesn't provide a nice install.bat for the entire process. Work done for #112 enabled remote download and execution of installers for POSIX and Windows and this work was re-usable for Windows installers hosted by pe_repo.

The pe_agent provisioner shipped in 0.14.0 will no longer fall back to pe_bootstrap for Windows and will instead configure pe_repo on the master and run installs from there.
